### PR TITLE
Stop adding old RPM GPG key 4172A230

### PR DESCRIPTION
**Pull request by bkabrda**
### What does this PR do?

Stops adding old RPM GPG key 4172A230

### Motivation

What inspired you to submit this pull request?

### Testing Guidelines

An overview on [testing](https://github.com/DataDog/dd-agent/blob/master/tests/README.md)
is available in our contribution guidelines.

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by amenasria** [comment]
CircleCI tests couldn't access the repo, I added a deploy key and rerun them. Waiting for them to pass but otherwise LGTM

File: packaging/datadog-agent/source/install_agent.sh [hunk 1]
```diff
@@ -41,6 +41,20 @@ solve your problem.\n\033[0m\n"
 }
 trap on_error ERR
 
+function remove_rpm_gpg_keys() {
+    local sudo_cmd="$1"
+    shift
+    local old_keys=("$@")
+    for key in "${old_keys[@]}"; do
+        if $sudo_cmd rpm -q "$key" 1>/dev/null 2>/dev/null; then
```
Comment (truthbk) [inline packaging/datadog-agent/source/install_agent.sh:49]: you might want to abbreviate with `>/dev/null 2>&1`, but this is fine.

**Review by truthbk — APPROVED** [review]
Looks good to me 👍

**Comment by truthbk** [comment]
> CircleCI tests couldn't access the repo, I added a deploy key and rerun them. Waiting for them to pass but otherwise LGTM

I don't think the CircleCI tests will pass either way, I believe they were already broken.

**Comment by amenasria** [comment]
@truthbk right, they're asking for manual input hence timing out, so we'll have to do without them I guess !

**Review by amenasria — APPROVED** [review]
LGTM

**Pull request merged**